source/kubernetes/create-app/index.html.md

```diff
@@ -0,0 +1,191 @@
+    secrets:
+      GH_TOKEN: ${{ secrets.GOVUK_CI_GITHUB_API_TOKEN }}
+```
+
+> **Note**
+> You may need to manually create a `v0` tag (e.g. against the most recent commit) in your
+> repository for the `release` workflow to pick up on and start automatically creating releases with
+> incremented versions.
+
+### Add deploy workflow
+
+Add a workflow invoking the shared deploy workflow template to `.github/workflows/deploy.yml`, for
+example:
+
+```yml
+name: Deploy
+
+run-name: Deploy ${{ inputs.gitRef || github.ref_name  }} to ${{ inputs.environment || 'integration' }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitRef:
+        description: 'Commit, tag or branch name to deploy'
+        required: true
+        type: string
+        default: 'main'
+      environment:
+        description: 'Environment to deploy to'
+        required: true
+        type: choice
+        options:
+        - integration
+        - staging
+        - production
+        default: 'integration'
+  release:
+    types: [released]
+
+jobs:
+  build-and-publish-image:
+    if: github.event_name == 'workflow_dispatch' || startsWith(github.ref_name, 'v')
+    name: Build and publish image
+    uses: alphagov/govuk-infrastructure/.github/workflows/build-and-push-image.yml@main
+    with:
+      gitRef: ${{ inputs.gitRef || github.ref_name }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_GOVUK_ECR_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_GOVUK_ECR_SECRET_ACCESS_KEY }}
+  trigger-deploy:
+    name: Trigger deploy to ${{ inputs.environment || 'integration' }}
+    needs: build-and-publish-image
+    uses: alphagov/govuk-infrastructure/.github/workflows/deploy.yml@main
+    with:
+      imageTag: ${{ needs.build-and-publish-image.outputs.imageTag }}
+      environment: ${{ inputs.environment || 'integration' }}
+    secrets:
+      WEBHOOK_TOKEN: ${{ secrets.GOVUK_ARGO_EVENTS_WEBHOOK_TOKEN }}
+      WEBHOOK_URL: ${{ secrets.GOVUK_ARGO_EVENTS_WEBHOOK_URL }}
+      GH_TOKEN: ${{ secrets.GOVUK_CI_GITHUB_API_TOKEN }}
+```
+
+## Add image tag files to `govuk-helm-charts`
+
+[GOV.UK Helm Charts](https://github.com/alphagov/govuk-helm-charts) uses image tag files to keep
+track of the image used for the active deployment in each environment. Add an initial tag file named
+after your application into the `image-tags` folder for each of the four environments, i.e.
+`charts/app-config/image-tags/{test,integration,staging/production}/example`:
+
+```
+image_tag: v1
+automatic_deploys_enabled: true
+promote_deployment: true
+```
+
+## Add initial secrets to AWS Secrets Manager
+
+### Sentry
+
+In order to set up Sentry for your app, you will need a _DSN_ from the Sentry UI. This must be
+manually added as a key/value pair to the `govuk/common/sentry` secret in all three environments.
+
+You can use `gds-cli aws govuk-{integration/staging/production}-poweruser -l` to log in to the AWS
+Console for each environment and make the change.
```

Comment:
I'd remove this, we don't need to tell people how to log into AWS in this document.